packages/protocol/contracts/team/airdrop/ERC20Airdrop2.sol

```diff
@@ -36,7 +36,7 @@ contract ERC20Airdrop2 is MerkleClaimable {
     /// @notice Event emitted when a user withdraws their tokens.
     /// @param user The address of the user.
     /// @param amount The amount of tokens withdrawn.
-    event Withdrawn(address user, uint256 amount);
+    event Withdrawn(address indexed user, uint256 amount);
```

Comment:
Dani, how about we delete this file.

Reply:
I'm assuming ERC20Airdrop is the one we'll use.

Reply:
Yes, true. I'm removing now.